tests/fixtures/analysis_form/analysis_form.json

```diff
@@ -8,7 +8,6 @@
     {
       "type": "transcript",
       "name": "record_a_note/transcript_en",
-      "label": "record_a_note Transcript (en)",
```

Comment:
To match KPI, transcripts and translations don't have labels in the analysis form definition